File: scoutingapp/src/main/java/org/usfirst/frc/team25/scouting/data/models/Release.java
package org.usfirst.frc.team25.scouting.data.models;

import lombok.Data;

/**
 * Created by sng on 2/18/2018.
 */

@Data
public class Release {

    private String tagName;
    private Asset[] assets;

    @Data
    public class Asset {
        String browserDownloadUrl;
        String name;
    }
}

File: scoutingapp/src/main/java/org/usfirst/frc/team25/scouting/data/models/Release.kt
package org.usfirst.frc.team25.scouting.data.models

/**
 * Created by sng on 2/18/2018.
 */
data class Release(val tagName: String,
                   val assets: Array<Asset>)

data class Asset(val browserDownloadUrl: String,
                 val name: String)